src/components/offerable/Offerable.tsx

```diff
@@ -0,0 +1,262 @@
+import clsx from 'clsx';
```

Comment:
Lets keep the offerable and buyable hook but not include the components for now.